man/src/network/create.md

```diff
@@ -117,3 +117,20 @@ By default, when you connect a container to an `overlay` network, Docker also
 connects a bridge network to it to provide external connectivity. If you want
 to create an externally isolated `overlay` network, you can specify the
 `--internal` option.
+
+### Network ingress mode
```

Comment:
oh, can you copy this example to the reference docs as well? (docs/reference/commandline/network_create.md)